CHANGELOG.rst

```diff
@@ -24,7 +24,6 @@ Fixed
 - incorrect atomic mass unit for xTB calculations
 
 
-
```

Comment:
Could you add the blank line back here and then squash the commits in this branch down to a single change?